app/domain/invoices/sac_memberships/member.rb

```diff
@@ -79,18 +79,8 @@ def paying_person?(beitragskategorie)
         !beitragskategorie.family? || sac_family_main_person?
       end
 
-      def family_members
-        person
-          .household_people
-          .includes(:roles)
-          .order(Person.order_by_name_statement)
-          .select { |p| sac_mitgliedschaft?(p) }
-      end
-
-      private
-
-      def sac_mitgliedschaft?(person)
-        People::SacMembership.new(person, date: date).active?
+      def household_people
+        person.household_people.select { |person| person.sac_membership_active? }
```

Comment:
Das `person.sac_membership_active?` berücksichtigt das `date` nicht sondern nimmt als Referenzdatum immer `Date.current`.
Es muss explizit eine `SacMembership` instanziert werden mit dem korrekten Referenzdatum, so wie das ja schon implementiert ist in `#sac_mitgliedschaft?(person)`, also einfach diese Methode verwenden.